File: qc-core/src/main/java/com/eager/questioncloud/core/domain/subscribe/event/SubscribedEvent.java
package com.eager.questioncloud.core.domain.subscribe.event;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class SubscribedEvent {
    private Long creatorId;

    public static SubscribedEvent create(Long creatorId) {
        return new SubscribedEvent(creatorId);
    }
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/subscribe/event/SubscribedEvent.kt
package com.eager.questioncloud.core.domain.subscribe.event

class SubscribedEvent(
    val creatorId: Long
) {
    companion object {
        @JvmStatic
        fun create(creatorId: Long): SubscribedEvent {
            return SubscribedEvent(creatorId)
        }
    }
}
